InternalDocs/exception_handling.md

```diff
@@ -0,0 +1,196 @@
+Description of exception handling
+---------------------------------
+
+Python uses a technique known as "zero-cost" exception handling, which
+minimizes the cost of supporting exceptions. In the common case (where
+no exception is raised) the cost is reduced to zero (or close to zero).
+The cost of raising an exception is increased, but not by much.
+
+The following code:
+
+```
+try:
+    g(0)
+except:
+    res = "fail"
+
+```
+
+compiles into pseudo-code like the following:
```

Comment:
"pseudo-code" usually means "approximately code". I think you have a more precise meaning of "bytecode with pseudo instructions". Is that right?

May be use "intermediate code" and describe `SETUP_FINALLY`, etc as "pseudo instruction"s?